app/src/main/proto/token_keys_data_store.proto

```diff
@@ -0,0 +1,9 @@
+syntax='proto3';
+
+option java_package = "org.p2p.wallet";
+option java_multiple_files = true;
+
+message TokenKeysDataStore {
+  string public_key = 1;
```

Comment:
encrypted_public_key
encrypted_private_key

can we make private_key into string too?